app/models/feedback.rb

```diff
@@ -0,0 +1,6 @@
+class Feedback < ApplicationRecord
+  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }, presence: true
```

Comment:
keep order in human readable way
`validates :email, presence: true, format: { ... }`
we are validatin' `email`, is required, and must match the format